orangegrove.rb

```diff
@@ -0,0 +1,85 @@
+# ORANGE TREE ------------------------------------------------------------------------------->
+class OrangeTree
+
```

Comment:
I've changed my settings. Thanks for letting me know about this. It's showing up a lot better on my current projects now.